src/components/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Footer from './Footer';
+
+describe('<Footer />', () => {
+  test('renders', () => {
+    const wrapper = shallow(<Footer />);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
We're not using snapshot testing any more, so this can be removed. See https://github.com/wellcometrust/corporate-components/blob/develop/src/components/SearchPane/SearchPane.test.tsx for an example of writing a simple mounting test.

Reply:
again haven't done this yet.

Reply:
Same for this - don't want to merge "unready" code in to develop really, and if this can be sorted then we can turn this in to a "proper" PR